tapalcatl_server/handler.go

```diff
@@ -116,6 +121,10 @@ func logBool(x bool) string {
 	}
 }
 
+func logDuration(x time.Duration) string {
+	return fmt.Sprintf("%d", x.Nanoseconds())
```

Comment:
No, just that I'm adding metrics for times like parsing the request and finding the format within the zip that's contained in memory. These take less time than a millisecond and that's why they are showing up as 0. They are always emitted now, even if 0, in f5611f5.